Makefile

```diff
@@ -9,24 +9,23 @@ else
 endif
 
 docker-compose := docker-compose
+docker := docker
 ifeq ($(detectedOS),Linux)
   docker-compose := sudo -E docker-compose
+  docker := sudo -E docker
 endif
 
-ifeq ($(container),)
-  container := web
-endif
+container ?= web
```

Comment:
Nice never knew about `?=`

Reply:
Yeah, when i was going through and removing those first things I found another reference that listed a bunch of cool things like this.